SMLHelper/Patchers/SurvivalPatcher.cs

```diff
@@ -0,0 +1,47 @@
+﻿#if SUBNAUTICA
+namespace SMLHelper.V2.Patchers
+{
+    using System;
+    using System.Collections.Generic;
+    using HarmonyLib;
+    using UnityEngine;
+    using Logger = Logger;
+
+    internal class SurvivalPatcher
+    {
+        internal static IDictionary<TechType, List<Action>> SurvivalDictionaryOnUse = new SelfCheckingDictionary<TechType, List<Action>>("SurvivalDictionaryOnUse", TechTypeExtensions.sTechTypeComparer);
+        internal static IDictionary<TechType, List<Action>> SurvivalDictionaryOnEat = new SelfCheckingDictionary<TechType, List<Action>>("SurvivalDictionaryOnEat", TechTypeExtensions.sTechTypeComparer);
+        internal static List<TechType> InventoryUseables = new List<TechType>();
+
+        internal static void Patch(Harmony harmony)
+        {
+            harmony.Patch(AccessTools.Method(typeof(Survival), nameof(Survival.Use)),
+                postfix: new HarmonyMethod(typeof(SurvivalPatcher), nameof(UsePostfix)));
+
+            harmony.Patch(AccessTools.Method(typeof(Survival), nameof(Survival.Eat)),
+                postfix: new HarmonyMethod(typeof(SurvivalPatcher), nameof(EatPostfix)));
```

Comment:
Both of these methods have the same signature.  
`UsePostFix` and `EatPostFix` could be combine into a single method and re-used for both of these postfix patches.